src/meetbot/features/introduction.ts

```diff
@@ -1,14 +1,16 @@
 import { Bot } from '..';
-import { postToChatJob } from '../pptr-helpers';
+import { postToChatJob } from '../meetbot-helpers';
+
+const GREETING_MESSAGE =
+	process.env.GREETING_MESSAGE ||
+	"Hello balenistas, it's your favorite bot, hubot!!";
```

Comment:
I think the default shouldn't be balena-specific

Reply:
I kept it like that to help deployments, but you are right. I will tweak it on the dashboard instead.